logstash-core/spec/logstash/compiler/compiler_spec.rb

```diff
@@ -0,0 +1,580 @@
+require "spec_helper"
+require "logstash/compiler"
+java_import Java::OrgLogstashConfigIr::DSL
+
+describe LogStash::Compiler do
+  def j
```

Comment:
This is used all over this file, so its here as shorthand to make the tests more readable, while still providing more namespacing safety than `import`.